### PR TITLE
minikube profile for save-demo

### PR DESCRIPTION
**Pull request by sanyavertolet**
This PR is a part of #1967 

### What's done:
 * Partially added minikube support in order to run save-demo outside from kubernetes but with an ability to use KubernetesService and so on
 * Updated liquibase container version in order to support arm architecture
 * Added load balancer services for local minikube deployment - just in case

### CODE REVIEW

**Comment by sanyavertolet** [comment]
@nulls take a look at what I have. Maybe you will have an idea. I get 403 when trying to download from Storage replacing `localhost` with `host.minikube.internal`. The point is that `netcat` says `host.minikube.internal:9000` is available from Minikube (checked with `minikube ssh`). Do you know if there are there any limitations from minio that are connected with hostname?

**Comment by nulls** [comment]
> @nulls take a look at what I have. Maybe you will have an idea. I get 403 when trying to download from Storage replacing `localhost` with `host.minikube.internal`. The point is that `netcat` says `host.minikube.internal:9000` is available from Minikube (checked with `minikube ssh`). Do you know if there are there any limitations from minio that are connected with hostname?

Yes, it is: presigned url is used in generating an url. So replacement should be done on configuration level to use this url in presigned url

File: buildSrc/src/main/kotlin/com/saveourtool/save/buildutils/DockerStackConfiguration.kt [hunk 1]
```diff
@@ -69,7 +69,7 @@ private fun Project.registerLiquibaseTask(projectName: String, relativeChangeLog
             "--rm",
             "--env", "INSTALL_MYSQL=true",
             "--network", "build_default",
-            "liquibase/liquibase:4.15",
+            "liquibase/liquibase:4.20",
```
Comment (nulls) [inline buildSrc/src/main/kotlin/com/saveourtool/save/buildutils/DockerStackConfiguration.kt:72]: what's about `liquibase/liquibase` in the rest deployments?

**Review by nulls — APPROVED** [review]

File: buildSrc/src/main/kotlin/com/saveourtool/save/buildutils/DockerStackConfiguration.kt (hunk 1 above)
Comment (sanyavertolet) [reply buildSrc/src/main/kotlin/com/saveourtool/save/buildutils/DockerStackConfiguration.kt:72]: There is no problem with the other deployment. Will change if really required

Comment (sanyavertolet) [reply buildSrc/src/main/kotlin/com/saveourtool/save/buildutils/DockerStackConfiguration.kt:72]: Done

**Pull request merged**